module-3/web-app/src/components/Pages/Projects/Projects.jsx

```diff
@@ -0,0 +1,33 @@
+import React, {useState, useEffect} from "react"
+import style from "./Projects.module.scss"
+
+//TODO in progress projects catalogue page
+const Projects = () => {
+    const [projects, setProjects] = useState('')
+   
+    useEffect(() => {
+        async function getData(url = "http://localhost:3001/selectAllProjects") {
+            const response = await fetch(url, {
+                method: 'GET'
+            })
+
+            return response.json() 
+        }
+        getData().then(res => setProjects(res))
+    }, [])
+     
+    return (
+        <div className={style.container}> 
+            {projects.length > 0 && projects.map((project, i) => (
+                <div className={style.project} key={i}>
```

Comment:
Sorry it's my fault, I think I told you to do it this way, but you can't actually use an array index as a react key prop. Use projectId instead. Please refer to this playlist for a deep dive explanation WHY https://www.youtube.com/watch?v=7YhdqIR2Yzo&list=PLxRVWC-K96b0ktvhd16l3xA6gncuGP7gJ

Reply:
Will be fixed in another pull request